packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -248,13 +278,18 @@ const TooltipTrigger = React.forwardRef<TooltipTriggerElement, TooltipTriggerPro
           aria-describedby={context.open ? context.contentId : undefined}
           data-state={context.stateAttribute}
           {...triggerProps}
-          ref={composedTriggerRef}
-          onPointerEnter={composeEventHandlers(props.onPointerEnter, (event) => {
+          ref={composedRefs}
+          onPointerMove={composeEventHandlers(props.onPointerMove, (event) => {
+            if (wasPointerDown.current) return;
```

Comment:
Needed to move `onPointerEnter` -> `onPointerMove` so that a tooltip opens if you're hovering a trigger as the grace area is dismissed. Would be easier to demo the behaviour but this check stops the tooltip from re-opening on move after it's been dismissed with `pointerDown`